sys/linux/dev_camx.txt.const

```diff
@@ -0,0 +1,45 @@
+# Code generated by syz-sysgen. DO NOT EDIT.
+arches = arm64
```

Comment:
You'll only need it on arm64, right?

Reply:
Yes. Should I have added the following line in the dev_cam.txt file as well?
```
meta arches["arm64"]
```

Reply:
If it even in general cannot get built/extracted on other arches, then yes, it'd be better to add it. Otherwise, I think it's fine as it is.

Reply:
Then, I'd like to leave it as it is. Currently it is only for arm64, but I confirmed that I can generate constants for arm64 as well. Thanks!